Products/ZenUI3/browser/resources/js/zenoss/devicemanagement/Administration.js

```diff
@@ -958,7 +960,8 @@ Ext.define("Zenoss.devicemanagement.Administration", {
                         sortable: true,
                         renderer: function(val){
                             var chunk = val.split(".");
-                            return chunk[0];
+                            var tz = val.split(" ")[2]
+                            return chunk[0]+" "+tz;
```

Comment:
OK. This is removing the microseconds portion of the original value, which looks like
> 2016/02/02 00:00:00.000 UTC